notifications/util.go

```diff
@@ -27,11 +33,25 @@ func toHeaders(headersArray []string) map[string]string {
 			headersMap[strings.TrimSpace(kv[0])] = ""
 			continue
 		}
-		headersMap[strings.TrimSpace(kv[0])] = strings.TrimSpace(kv[1])
+		headersMap[strings.TrimSpace(kv[0])] = ReplaceEnvVariablePlaceholder(strings.TrimSpace(kv[1]))
 	}
 	return headersMap
 }
 
+func ReplaceEnvVariablePlaceholder(value string) string {
```

Comment:
Minor: in Go, functions starting with an upper case letter are "public". Do you need this function to be visible outside of this package?

Reply:
Good point, I don't need this visibility, fixed with 8eba014